pyuvdata/tests/test_uvdata.py

```diff
@@ -1548,3 +1548,160 @@ def test_get_feedpols():
+    nt.assert_is_instance(ant_pairs_nums,type(None))
+    nt.assert_items_equal(polarizations,polarizations_data)
+
+    # Unparsible string
+    ant_str = 'none'
+    nt.assert_raises(ValueError, uv.parse_ants, ant_str)
+
+    # Single antenna number
+    ant_str = '0'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(0, 0),(0, 1),(0, 2),(0, 3),(0, 6),(0, 7),(0, 8),(0, 11),
+                               (0, 14),(0, 18),(0, 19),(0, 20),(0, 21),(0, 22),
+                               (0, 23),(0, 24),(0, 26),(0, 27)]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_is_instance(polarizations,type(None))
+
+    # Multiple antenna numbers as list
+    ant_str = '22,26'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(0, 22),(0, 26),(1, 22),(1, 26),(2, 22),(2, 26),(3, 22),(3, 26),(6, 22),
+                               (6, 26),(7, 22),(7, 26),(8, 22),(8, 26),(11, 22),(11, 26),(14, 22),
+                               (14, 26),(18, 22),(18, 26),(19, 22),(19, 26),(20, 22),(20, 26),(21, 22),
+                               (21, 26),(22, 22),(22, 23),(22, 24),(22, 26),(22, 27),(23, 26),(24, 26),
+                               (26, 26),(26, 27)]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_is_instance(polarizations,type(None))
+
+    # Single baseline
+    ant_str = '1_3'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(1,3)]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_is_instance(polarizations,type(None))
+
+    # Single baseline with polarization
+    ant_str = '1l_3r'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(1, 3)]
+    polarizations_data = [-4]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_items_equal(polarizations,polarizations_data)
+
+    # Single baseline with single polarization in first entry
+    ant_str = '1l_3,2x_3'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(1, 3), (2, 3)]
+    polarizations_data = [-2, -4, -5, -7]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_items_equal(polarizations,polarizations_data)
+
+    # Single baseline with single polarization in last entry
+    ant_str = '1_3l,2_3x'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(1, 3),(2, 3)]
+    polarizations_data = [-2, -3, -5, -8]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_items_equal(polarizations,polarizations_data)
+
+    # Multiple baselines as list
+    ant_str = '1_2,1_3,1_10'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(1, 2),(1, 3),(1, 10)]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_is_instance(polarizations,type(None))
+
+    # Multiples baselines with polarizations as list
+    ant_str = '1r_2l,1l_3l,1r_11r'
```

Comment:
We should make it clear in the documentation that if you pass the result from a selection like this, you don't just get the three baselines, but rather you get all the pairs of antennas with all the polarizations. In this case you end up with:
1r_2l, 1l_2l, 1r_2r, 1r_3l, 1l_3l, 1r_3r, 1r_11l, 1l_11l, and 1r_11r.
If the intent is to pass to select, this is the right thing to do, but it should be clear.